examples/module-federation/mf-react-component/stories/index.stories.tsx

```diff
@@ -0,0 +1,11 @@
+// @ts-ignore ignore remote module type check for passing ci run build because we set @mf-types folder in .gitignore
```

Comment:
this is not an expect error, in local we have @mf-types for type check. So if we change this to expect error, we will receive error hint to remove this comment, this can not be changed

Reply:
So is it an TS error? If not, why there's an `@ts-ignore`?

Reply:
yes, you can consider this is a ts error in ci env because we set @mf-types into .gitignore, when storybook build, type check will throw error

Reply:
How to generate in CI env just like local.

Reply:
remove @mf-types from `.gitignore`

Reply:
like this folder https://github.com/module-federation/core/pull/3131/files#diff-cf94c81add15f5e791dc7eb01c930f05f645750b1f6fda34ce6727b463f22fb0

Reply:
files in this folder are auto generated by mf